OWNERS.md

```diff
@@ -0,0 +1,14 @@
+# OWNERS
+
+This page lists all maintainers for **this** repository. Each repository in the [Crossplane
+organization](https://github.com/crossplane/) will list their repository maintainers in their own
+`OWNERS.md` file.
+
+Please see the Crossplane
+[GOVERNANCE.md](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md) for governance
+guidelines and responsibilities for the steering committee and maintainers.
+
+## Maintainers
+
+* Maintainer 1 <maintainer1@acme.org> ([maintainer1](https://github.com/maintainer1))
```

Comment:
do you want to go ahead and add yourself as a maintainer on this project @3pings? or other folks that will also be contributing and should have merge permissions?